docs/modules/ROOT/pages/how-tos/exoscale/remove_storage_node.adoc

```diff
@@ -0,0 +1,243 @@
++
+[source,bash]
+----
+commodore catalog compile "${CLUSTER_ID}"
+----
+
+== Set alert silence
+
+:duration: +60 minutes
+include::partial$create-alertmanager-silence.adoc[]
+
+== Update Cluster Config
+
+. Update cluster config. 
++
+[source,bash]
+----
+pushd "inventory/classes/${TENANT_ID}/"
+
+yq eval -i ".parameters.openshift4_terraform.terraform_variables.storage_count -= 1" \
+  ${CLUSTER_ID}.yml
+
+yq eval -i ".parameters.rook_ceph.ceph_cluster.node_count -= 1" \
+  ${CLUSTER_ID}.yml
+
+----
++
+[NOTE]
+====
+Ceph can't scale below 3 storage nodes, which is the default number of nodes.
+Please check that this update doesn't reduce the number of storage nodes below 3, before continuing.
+====
+
+. Review and commit
++
+[source,bash]
+----
+
+# Have a look at the file ${CLUSTER_ID}.yml.
+
+git commit -a -m "Remove storage node to cluster ${CLUSTER_ID}"
+git push
+
+popd
+----
+
+. Compile and push cluster catalog 
++
+[source,bash]
+----
+commodore catalog compile ${CLUSTER_ID} --push -i
+----
+
+== Prepare Terraform environment
+
+include::partial$exoscale/configure-terraform-secrets.adoc[]
+
+include::partial$setup_terraform.adoc[]
+
+== Remove Node
+Make a note of the node you want to remove
+
+[source,bash]
+----
+export NODE_TO_REMOVE=storage-XXXX
+----
+
+=== Remove old OSD
+. Make sure ArgoCD ran and reduced the target number of OSDs
++
+[source,bash]
+----
+kubectl --as=cluster-admin -n syn-rook-ceph-cluster \
+  get cephcluster cluster -o jsonpath='{.spec.storage.storageClassDeviceSets[0].count}'
+----
+
+:osd-node: ${NODE_TO_REMOVE}
+include::partial$storage-ceph-remove-osd.adoc[]
+
+. Make a note of the PVC of the old OSD
++
+NOTE: We also extract the name of the PV here, but we'll only delete the PV after removing the node from the cluster.
++
+[source,bash]
+----
+pvc_name=$(kubectl --as=cluster-admin -n syn-rook-ceph-cluster get deploy \
+  "rook-ceph-osd-${OSD_ID}" -ojsonpath='{.metadata.labels.ceph\.rook\.io/pvc}')
+pv_name=$(kubectl --as=cluster-admin -n syn-rook-ceph-cluster get pvc \
+  "${pvc_name}" -o jsonpath='{.spec.volumeName}')
+----
+
+. Check if the OSD deployment needs to be deleted, and delete it if necessary
++
+[source,bash]
+----
+kubectl --as=cluster-admin -n syn-rook-ceph-cluster get deploy \
+  -l failure-domain="${NODE_TO_REMOVE}"
+# Run this command if the previous command lists a deployment
+kubectl --as=cluster-admin -n syn-rook-ceph-cluster delete deploy \
+  -l failure-domain="${NODE_TO_REMOVE}"
+----
+
+. Clean up PVC and prepare job of the old OSD if necessary
++
+[source,bash]
+----
+kubectl --as=cluster-admin -n syn-rook-ceph-cluster delete job \
+  -l ceph.rook.io/pvc="${pvc_name}"
+kubectl --as=cluster-admin -n syn-rook-ceph-cluster delete pvc "${pvc_name}"
+----
+
+. Clean up PVC encryption secret
++
+[source,bash]
+----
+kubectl --as=cluster-admin -n syn-rook-ceph-cluster delete secret -l pvc_name="${pvc_name}"
+----
```

Comment:
Either add this to the `storage-ceph-remove-osd.adoc` partial (including scaling the operator step (conditional on another attribute) which is omitted here), or create another partial for this part, as it seems identical to the steps for replacing a node.